test/e2e/gitops_service_test.go

```diff
@@ -633,33 +741,65 @@ func validateClusterConfigChange(t *testing.T) {
 
 }
 
-func applyMissingPermissions(f *framework.Framework, ocPath string) error {
-
+func applyMissingPermissions(ctx *framework.Context, f *framework.Framework, name, namespace string) error {
```

Comment:
Had to refactor this function to create roles in other namespaces. Ref: [Issue #148 ](https://github.com/redhat-developer/gitops-operator/issues/148)